Modules/TPC/include/TPC/CheckOfTrendings.h

```diff
@@ -17,6 +17,8 @@
 #define QC_MODULE_TPC_CHECKOFTRENDINGS_H
 
 #include "QualityControl/CheckInterface.h"
+#include <TGraph.h>
+#include <TCanvas.h>
```

Comment:
Forward declare:

```suggestion
class TGraph;
class TCanvas;
```

And move
```
#include <TGraph.h>
#include <TCanvas.h>
```
to the `cxx`